editor/src/components/canvas/controls/grid-controls.tsx

```diff
@@ -2859,6 +2859,21 @@ const RulerMarkerIndicator = React.memo(
 
     const labelClass = 'ruler-marker-label'
 
+    const edge: GridResizeEdge =
```

Comment:
Could this be made into: `${props.axis}-${props.marker.bound}`?

Reply:
oh, very nice idea! I did not even know TS would allow that without complaining about types! ✅